buildpacks/homebrew.go

```diff
@@ -44,15 +61,51 @@ func (bt HomebrewBuildTool) InstallDir() string {
 func (bt HomebrewBuildTool) Install() error {
 	gi := goInfo.GetInfo()
 
+	var err error
 	switch gi.GoOS {
 	case "darwin":
-		return bt.InstallDarwin()
+		err = bt.InstallDarwin()
 	case "linux":
-		return bt.InstallLinux()
+		err = bt.InstallLinux()
 	default:
-		fmt.Printf("Unsupported platform: %s\n", gi.GoOS)
-		return nil
+		err = fmt.Errorf("Unsupported platform: %s\n", gi.GoOS)
```

Comment:
no need for newline in Errorf, same below

Reply:
..and better not have it

Reply:
Fixed -- conversion from Printf artifact